.github/workflows/test-gcp-dm.yml

```diff
@@ -80,7 +82,7 @@ jobs:
 
       - name: Install CSPM GCP integration
         id: cspm-gcp-integration
-        working-directory: deploy/test-environments/fleet_api
+        working-directory: tests/integrations_setup
```

Comment:
i didn't see there is a PR for this already so opened one too https://github.com/elastic/cloudbeat/pull/2125/files
let me know if you want to use this one or the other one
in anyway -
1. i think `poetry install` will fail as the `toml` file is not in `tests/integrations_setup`
2. i think `poetry run python src/install_cspm_gcp_integration.py` will fail as the working dir has no `src`